tests/Unit/Helpers/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/TestHelpers.hpp

```diff
@@ -0,0 +1,111 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Framework/TestingFramework.hpp"
+
+#include <memory>
+#include <pup.h>
+#include <string>
+#include <tuple>
+#include <utility>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/DampingFunction.hpp"
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "Parallel/PupStlCpp11.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/Literals.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/Overloader.hpp"
+
+namespace TestHelpers::GeneralizedHarmonic::ConstraintDamping {
+namespace detail {
+template <size_t VolumeDim, typename Fr, class... MemberArgs, class T>
+void check_impl(
+    const std::unique_ptr<
+        ::GeneralizedHarmonic::ConstraintDamping::DampingFunction<
+            VolumeDim, Fr>>& in_gh_damping_function,
+    const std::string& python_function_prefix, const T& used_for_size,
+    const std::array<std::pair<double, double>, 1> random_value_bounds,
+    const MemberArgs&... member_args) noexcept {
+  using GhDampingFunc =
+      ::GeneralizedHarmonic::ConstraintDamping::DampingFunction<VolumeDim,
+                                                                  Fr>;
+  using CallOperatorFunction =
+      Scalar<T> (GhDampingFunc::*)(const tnsr::I<T, VolumeDim, Fr>&)
+          const noexcept;
+
+  const auto member_args_tuple = std::make_tuple(member_args...);
+  const auto helper =
+      [&python_function_prefix, &random_value_bounds, &member_args_tuple,
+       &used_for_size](
+          const std::unique_ptr<GhDampingFunc>& gh_damping_function) noexcept {
+        // need func variable to work around GCC bug
+        CallOperatorFunction func{&GhDampingFunc::operator()};
+
+        INFO("Testing call operator...")
+        pypp::check_with_random_values<1>(
+            func, *gh_damping_function, "TestFunctions",
+            python_function_prefix + "_call_operator", random_value_bounds,
+            member_args_tuple, used_for_size);
+        INFO("Done testing call operator...")
+        INFO("Done\n\n")
+      };
+  helper(in_gh_damping_function);
+  helper(serialize_and_deserialize(in_gh_damping_function));
+}
+}  // namespace detail
+// @{
+/*!
+ * \ingroup TestingFrameworkGroup
+ * \brief Test a DampingFunction by comparing to python functions
+ *
+ * The python functions must be added to TestFunctions.py in
+ * tests/Unit/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/Python.
+ * The prefix for each class of DampingFunction is arbitrary, but should
+ * generally be descriptive (e.g. 'gaussian_plus_constant') of the
+ * DampingFunction.
+ *
+ * The `python_function_prefix` argument passed to `check` must be `PREFIX`. If
```

Comment:
I'm not sure what this sentence means. Where is uppercase PREFIX defined?

Reply:
Again, this goes back to #625, and the same comment migrated to `TestHelpers.hpp` in MathFunctions and now ConstraintDamping. I take it to mean that when you call the `check()` function, the `python_function_prefix` argument must be the same prefix used in the python file, as described in the previous paragraph. 

I'm happy to change this to something else, if you have a suggested improvement to the wording.

Reply:
Remove the "PREFIX", explain that this groups the python functions...every python function must start with this prefix